src/compiler/scala/tools/nsc/javac/JavaParsers.scala

```diff
@@ -894,10 +939,10 @@ trait JavaParsers extends ast.parser.ParsersCommon with JavaScanners {
       var pos = in.currentPos
       val pkg: RefTree =
         if (in.token == AT || in.token == PACKAGE) {
-          annotations()
+          annotations() // TODO: put these somewhere?
```

Comment:
Indeed. The fact that this gets called if `in.token == PACKAGE` seems surprising, as well.